docs/index.mdx

```diff
@@ -16,27 +16,46 @@ SCS is built, backed, and operated by an active open-source community worldwide.
 
 ## Use Cases and Deployment Examples
 
-### IaaS Layer
+### Virtualization (IaaS) Layer
+
+The SCS IaaS reference implementation is based on [OSISM](https://osism.tech/).
 
 #### Quick Start with Cloud-In-A-Box
 
-The fastest way to get in touch with SCS is to deploy a SCS cloud virtually. The Cloud-In-A-Box was built explicitly for this scenario. Check it out [here](/docs/iaas/guides/deploy-guide/examples/cloud-in-a-box)
+You can do a single node installation for learning, testing or development purposes.
+The Cloud-In-A-Box configuration was built explicitly for this scenario.
```

Comment:
Cloud in a Box

Reply:
Hmm, while I agree `in a` being lowercase, I consider the term Cloud-in-a-Box a single term rather than four words ... thus my preference to leave the hyphens in it. I believe we have used the form with hyphens at most places in the past. If we want to change that, we should probably grep through our repos and address it everywhere ...

Reply:
Partially addressed by 9fc2e2fe.